worlds/celeste64/Rules.py

```diff
@@ -0,0 +1,108 @@
+from typing import Callable
+
+from BaseClasses import CollectionState, MultiWorld
+from .Names import ItemName, LocationName
+from worlds.AutoWorld import World
+from worlds.generic.Rules import add_rule, set_rule, CollectionRule
+
+
+
+def set_rules(world: World):
```

Comment:
```suggestion
from typing import TYPE_CHECKING
if TYPE_CHECKING:
    from . import Celeste64World


def set_rules(world: "Celeste64World"):
```
Would recommend doing this to get the typing right, also cleaning up the unused imports

Reply:
It is my understanding that this is the standard. Feel free to do what you want